packages/jest-dom-mocks/src/connection.ts

```diff
@@ -0,0 +1,55 @@
+import set from 'lodash/set';
```

Comment:
`lodash/set` should be included in `package.json`

Reply:
oh weird, it's used in other ones but isn't there. I'll add it

Reply:
Actually I'm going to try just removing it instead

Reply:
I am not surprise actually, @cartogram told me there is actually A LOT of missing dependencies in quilt when he was working on an automatic way of checking this.

so I am trying to be more carful recently.